ipfx/ephys_data_set.py

```diff
@@ -115,20 +114,22 @@ def sweep(self, sweep_number):
         sweep_meta_data = self.get_sweep_meta_data(sweep_number)
         sampling_rate = sweep_data['sampling_rate']
         dt = 1. / sampling_rate
-        sweep_start_ix, sweep_end_ix = ep.get_sweep_epoch(sweep_data['response'])
+        t = np.arange(0, len(sweep_data['stimulus'])) * dt
+        assert len(sweep_data['stimulus']) == len(sweep_data['response'])
```

Comment:
Are there cases where `len(sweep_data['stimulus']) != len(sweep_data['response'])`? If so, it should probably be handled? If not, is there any point to the assertion?

Reply:
I removed this check altogether for now.